### PR TITLE
Add `GBC 2022 Autumn` tournament page 

### PR DESCRIPTION
**Pull request by RushFTK**
Co-authored-by: @yobrevelc

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)

### CODE REVIEW

**Comment by RushFTK** [comment]
All CI fix commit could be rebase into ones. thanks for help.

File: wiki/Tournaments/GBC/GBC_2022_Autumn/en.md [hunk 1]
```diff
@@ -0,0 +1,267 @@
...
+
+# GB Cup 2022 Autumn
+
+![GBC 2022 Autumn banner](img/GBC-2022-Autumn-banner.png)
+
+The **GB Cup 2022 Autumn** (***GBC 2022 Autumn***) is a 3v3 captain draft-based osu!mania 4-key tournament hosted by Team GB. It is the second instalment of the GB Cup.
+
+## Tournament schedule
+
+*this might change depending on the number of incoming registrations.*
```
Comment (TPGPL) [inline wiki/Tournaments/GBC/GBC_2022_Autumn/en.md:17]:
```suggestion
*The schedule might change depending on the number of incoming registrations.*
```

File: wiki/Tournaments/GBC/GBC_2022_Autumn/en.md [hunk 2]
```diff
@@ -0,0 +1,267 @@
...
+| Regular stage week 3 | 2022-11-12/2022-11-13 |
+| Regular stage week 4 | 2022-11-19/2022-11-20 |
+| Quarterfinals | 2022-11-26/2022-11-27 |
+| Semifinals | 2022-12-03/2022-12-04 |
+| Finals | 2022-12-10/2022-12-11 |
+
+## Prizes
+
+| Placing | Prizes |
+| :-----: | :----- |
```
Comment (TPGPL) [inline wiki/Tournaments/GBC/GBC_2022_Autumn/en.md:37]:
```suggestion
| :-: | :-- |
```

File: wiki/Tournaments/GBC/GBC_2022_Autumn/en.md [hunk 3]
```diff
@@ -0,0 +1,267 @@
...
+| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | Profile badge, KanonBot profile badge, *TBA* CNY |
+| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | KanonBot profile badge, *TBA* CNY |
+| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | KanonBot profile badge, *TBA* CNY |
+| 4th place | *TBA* CNY |
+| 5-8th place | *TBA* CNY |
```
Comment (TPGPL) [inline wiki/Tournaments/GBC/GBC_2022_Autumn/en.md:42]:
```suggestion
| *4th place* | *TBA* CNY |
| *5-8th place* | *TBA* CNY |
```

**Review by TPGPL — CHANGES_REQUESTED** [review]

**Review by TPGPL — APPROVED** [review]

**Pull request merged**